### PR TITLE
PR annotation test

### PR DESCRIPTION
**Pull request by mwz**

### CODE REVIEW

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 1]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
       }
   }
 
+  def ExampleIssues(): Unit = {
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: info Method name not recommended ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Method name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.MethodNamesChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.MethodNamesChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 2]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+    val isEmpty = List(1).size == 0
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: info List.size is O(n) ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.ListSize&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.ListSize))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Avoid Traversable.size == 0, use Traversable.isEmpty instead ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.AvoidSizeEqualsZero&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.AvoidSizeEqualsZero))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Literal passed as argument without name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.NamedArgumentChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.NamedArgumentChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 3]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+    val b = if (true) true else false
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: info Unnecessary if condition. ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.UnnecessaryIf&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.UnnecessaryIf))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Constant if expression ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.ConstantIf&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.ConstantIf))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor No redundant if expressions ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.RedundantIfChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.RedundantIfChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 4]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+    var TWO = 1 + 1
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Var could be val ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.VarCouldBeVal&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.VarCouldBeVal))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: major No mutable local variables ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.VarLocalChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.VarLocalChecker))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Field name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.FieldNamesChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.FieldNamesChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 5]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+    //test
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Space after the start of the comment ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.SpaceAfterCommentStartChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.SpaceAfterCommentStartChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 6]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+    while (true) {
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor While true loop ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.controlflow.WhileTrue&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.controlflow.WhileTrue))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor No while loops ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.WhileChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.WhileChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 7]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+      println("WOW!")
+    }
+    return ()
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Return ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.ReturnChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.ReturnChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 8]
```diff
@@ -74,6 +74,13 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
       }
   }
 
+  def ExampleIssues(): Unit = {
+    var TWO = 1 + 1
+    //test
+    val stillTwo = TWO / 1
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Divide by one ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.math.DivideByOne&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.math.DivideByOne))

File: src/main/scala/com/mwz/sonar/scala/util/MoreIssues.scala [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * You should have received a copy of the GNU General Lesser Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.mwz.sonar.scala
+package util
+
+object MoreIssues {
+  def MoreIssues = ???
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/util/MoreIssues.scala]: info Method name not recommended ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/util/MoreIssues.scala]: minor Method name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.MethodNamesChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.MethodNamesChecker))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/util/MoreIssues.scala]: minor Usage of ??? operator ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.NotImplementedErrorUsage&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.NotImplementedErrorUsage))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/util/MoreIssues.scala]: minor Public method must have type ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.PublicMethodsHaveTypeChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.PublicMethodsHaveTypeChecker))

**Comment by mwz** [comment]
All good 👍

**Pull request closed without merging**